pkg/control/controlexecute/execution_tree.go

```diff
@@ -56,14 +61,33 @@ func NewExecutionTree(ctx context.Context, workspace *workspace.Workspace, clien
 		return nil, err
 	}
 
-	// now identify the root item of the control list
-	rootItem, err := executionTree.getExecutionRootFromArg(arg)
-	if err != nil {
-		return nil, err
-	}
+	var resolvedItem modconfig.ModTreeItem
 
+	// if only one argument is provided, fall back to the original data structure
+	// tree -> root -> item
+	if len(args) == 1 {
+		resolvedItem, err = executionTree.getExecutionRootFromArg(args[0])
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		// for multiple items, use a virtual benchmark as the parent of the items
```

Comment:
let's call it a root benchmark - its not virtual it's really, it exists in the tree and has children